baselines/tamuna/pyproject.toml

```diff
@@ -0,0 +1,141 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "tamuna"
+version = "1.0.0"
+description = "Flower Baselines"
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.dev>"]
```

Comment:
I thought you might want to add your details to this `authors` field. Similarly, please update the `description` field above with just "tamuna" or maybe the full title of the Tamuna paper?

Reply:
added